metatron/src/test/java/uk/gov/ida/eidas/metatron/apprule/rules/CreatesConfigDropwizardClientRule.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.ida.eidas.metatron.apprule.rules;
+
+import io.dropwizard.testing.junit.DropwizardClientRule;
+import org.junit.Assert;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.net.URI;
+
+public class CreatesConfigDropwizardClientRule extends DropwizardClientRule {
+    public final static String CONFIG_LOCATION = "/tmp/countriesConfig.yaml";
+
+    public CreatesConfigDropwizardClientRule(MetatronTests.TestMetadataResource testMetadataResource) {
+        super(testMetadataResource);
+        try {
+            this.before();
```

Comment:
Did you find out what `this.before` was doing?

Reply:
The port was coming back as 0 unless I used that.